P/PMC/build_tarballs.jl

```diff
@@ -0,0 +1,33 @@
+using BinaryBuilder
+
+name = "LibPMC"
```

Comment:
This name must match exactly the name of the diretory

Reply:
of the directory in `Yggdrasil`?

Reply:
Yes, the directory is called `PMC, the package is called `LibPMC`, they should match up to the case.  Also, please read the documentation: https://docs.binarybuilder.org/stable/building/#Name